api/sql/schema/750$rule.limit.get.sql

```diff
@@ -0,0 +1,31 @@
+CREATE PROCEDURE [rule].[limit.get] --retrieve the rule limits set per user role, operation and currency
+    @channelId BIGINT, --actor id of the channel performing the operation
+    @operation VARCHAR(100) = 'loanApplicationApprove', --operation executed
+    @currency VARCHAR(3) = 'USD', --limit currency
+    @property NVARCHAR (50) = 'loanApprovalLevel' -- condition property
+AS
+DECLARE @operationId BIGINT = (
+    SELECT n.itemNameId
+    FROM [core].[itemName] n
+    JOIN [core].[itemType] t ON n.itemTypeId = t.itemTypeId AND t.alias = 'operation'
+    WHERE itemCode = @operation)
+
+DECLARE @userRoles AS TABLE (factor CHAR(2), actorId BIGINT)
+
+INSERT INTO @userRoles (factor, actorId)
```

Comment:
Is this variable really needed? Why not JOIN with actorRelatedRoles in the next select?